### PR TITLE
Add integration test for using URIs to connect to alluxio

### PR DESCRIPTION
**Pull request by LuQQiu**
Add integration tests for using URIs with connect details in authorities to connect to
alluxio cluster through hadoop filesystem

### CODE REVIEW

**Comment by LuQQiu** [comment]
@aaudiber Added the uri connection integration test. PTAL, thanks

File: tests/src/test/java/alluxio/client/hadoop/FileSystemUriIntegrationTest.java [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+import java.net.URI;
+
+/**
+ * Integration tests for using URIs with connect details in authorities to connect to
+ * Alluxio clusters through {@link FileSystem}.
+ */
+public class FileSystemUriIntegrationTest extends BaseIntegrationTest {
+  private static final int WAIT_TIMEOUT_MS = 60 * Constants.SECOND_MS;
+  public MultiProcessCluster mCluster;
```
Comment (aaudiber) [inline tests/src/test/java/alluxio/client/hadoop/FileSystemUriIntegrationTest.java]: private

File: tests/src/test/java/alluxio/client/hadoop/FileSystemUriIntegrationTest.java [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+  @After
+  public void after() throws Exception {
+    mCluster.destroy();
+  }
+
+  @Test
+  public void zookeeperUriTest() throws Exception {
```
Comment (aaudiber) [inline tests/src/test/java/alluxio/client/hadoop/FileSystemUriIntegrationTest.java]: can we factor out the commonality between these tests?

**Comment by LuQQiu** [comment]
Jenkins, test this please

**Comment by aaudiber** [comment]
alluxio-bot, merge this please

**Pull request merged**